File: ex1/answers/problem1.c
#include <math.h>
#include <stdio.h>

double
basel_sum (int n)
{
  double result = 0;
  int i;
  for (i=1;i<=n;i++) {
    result += 1/((double)(i*i));
  }
  return result;
}


int
main (void)
{
  printf("%f\n", sqrt(6*basel_sum(10000)));
}

File: ex1/answers/problem1.py
#!/usr/bin/env python

def basel_sum(n):
    result = 0
    for i in range(1,n+1):
        result += 1/float(i*i)
    return result

def main():
    print("{:f}".format((6*basel_sum(10000))**(0.5)))

if __name__ == "__main__":
    main()
